docs/source/quick_start.rst

```diff
@@ -0,0 +1,181 @@
+Quick Start
+============
+
+This guide is intended to jump start new Kokkos users (and beginners, in particular).  Kokkos Core is the foundation for all other Kokkos Ecosystem projects, and is readily usable in your own for performance portability.
+
+
+Download Latest and Build 
+-----------------------------
+
+.. note::
+
+  Please become familiar with `Kokkos Requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_, and verify that your machine has all necessary compilers, backend SDK and build system components.
+
+
+..
+ Nota bene:  the link for "Latest" should be stable from one release to the next, but check periodically to be sure 
+..
+
+:bdg-link-primary:`Latest Release <https://github.com/kokkos/kokkos/releases/latest>`
+
+.. code-block:: sh
+  
+  # Uncomment according to the type of file you've downloaded (zip or tar)
+  unzip kokkos-x.y.z.zip 
+  # tar -xzf kokkos-x.y.z.tar.gz
+  cd kokkos-x.y.z
+  mkdir build
+  cd build
+
+
+Basic Configuration
+-------------------
+
+.. note::
+
+  You can create small shell scripts to manage and experiment with configuration details, following the GPU microarchitecture-appropriate examples below.  Upon successful configuration, ``make -j`` to build, and ``make install`` to install.
```

Comment:
```suggestion
  You can create small shell scripts to manage and experiment with configuration details, following the GPU microarchitecture-appropriate examples below. 
```
It is better to use `cmake` to run compile and install. Default generator might not be `make` but `ninja` or other stuff on windows.

Reply:
Following the same logic, that the "Quick Start" is for new users and/or people with little experience, I want to clearly separate configure and build steps, and on Linux-like systems, I *believe* one would mostly use `make`.  If you wanted to provide recipes for Windows / MSVC, I'd be happy to include those, I'm just not well versed in Windows as a software development environment.